torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -247,6 +247,23 @@ def call_function(self, target, args, kwargs):
         ############################################################
         # Data movement
         ############################################################
+        if target == torch.ops.aten.col2im.default:
+            #FIXME(jdh8): how to handle kernel_size?
+            tensor, output_size, kernel_size, dilation, padding, stride = args
```

Comment:
I'm not sure how do we use `kernel_size` here.  Maybe `ttnn.fold` can infer that.